cfme/tests/containers/test_views_objects.py

```diff
@@ -66,8 +74,8 @@ def test_nodes_views():
     assert tb.is_active('List View'), "Nodes list view setting failed"
 
 
-def test_images_views():
-    sel.force_navigate('containers_images')
+def test_images_views(image):
+    navigate_to(image, 'All')
```

Comment:
Stuck in a mindset of needing an instance to navigate to a specific image, no need here.